docs/source/plugins.rst

```diff
@@ -0,0 +1,158 @@
+.. _xcube.core.gen.iproc.DefaultInputProcessor: https://github.com/dcs4cop/xcube/blob/master/xcube/core/gen/iproc.py
+.. _setup.py: https://github.com/dcs4cop/xcube/blob/master/setup.py
+.. _`Creating and discovering plugins`: https://packaging.python.org/guides/creating-and-discovering-plugins/
+.. _`Python Packing User Guide`: https://packaging.python.org/
+.. _`Dynamic Discovery of Services and Plugins`: https://setuptools.readthedocs.io/en/latest/setuptools.html#dynamic-discovery-of-services-and-plugins
+.. _`Setuptools documentation`: https://setuptools.readthedocs.io/
+.. _`Setuptools`: https://setuptools.readthedocs.io/
+.. _`SENTINEL Hub Cloud API`: https://www.sentinel-hub.com/
+.. _xcube_sh: https://github.com/dcs4cop/xcube-sh
+.. _xcube_gen_bc: https://github.com/dcs4cop/xcube-gen-bc
+.. _xcube_gen_rbins: https://github.com/dcs4cop/xcube-gen-rbins
+.. _xcube_gen_vito: https://github.com/dcs4cop/xcube-gen-vito
+
+=======
+Plugins
+=======
+
+xcube's functionality can be extended by plugins. A plugin is a Python module that is detected
+by xcube at runtime and dynamically loaded by xcube on demand, that is, once the extensions
+added by plugins need to be made available.
```

Comment:
I do not understand this sentence, is it possible to do a different wording?

Reply:
Yes. Reworded!